From 670f91c0a6d11bb6f891b95ddea7acb9fc890911 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Christian=20K=C3=BCndig?= <christian@kuendig.info>
Date: Fri, 12 Sep 2025 20:42:40 +0200
Subject: [PATCH] Update SDL3 port to version 3.2.22

---
 tools/ports/sdl3.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/ports/sdl3.py b/tools/ports/sdl3.py
index caf3fa8a5..1e164ce76 100644
--- a/tools/ports/sdl3.py
+++ b/tools/ports/sdl3.py
@@ -9,9 +9,9 @@ import shutil
 
 from tools import diagnostics
 
-VERSION = '3.2.4'
+VERSION = '3.2.22'
 TAG = f'release-{VERSION}'
-HASH = 'c26a8afeec481e3ae3b435eec405d9f99d78752ebf5118963cd56728ceff23772769f5291df581329488da7489034e835301b08d61a42c811764e24b3542a4c2'
+HASH = '2d49f43f37b681b3b12918518fc2bb89fd79f64b6f9592cceb504402a9cfb3d5c0ae6f437cdbcd8fdef11065dfb15a86fcb61a58cf1fa41af71a61f767ab7260'
 SUBDIR = f'SDL-{TAG}'
 
 variants = {'sdl3-mt': {'PTHREADS': 1}}
-- 
2.46.0

